### PR TITLE
Add chart support for observability queries

### PR DESCRIPTION
**Pull request by ebadyano**
Observability queries do't have target throughput and instead have `workflow-interval`

### CODE REVIEW

File: esrally/chart_generator.py [hunk 1]
```diff
@@ -1666,7 +1666,8 @@ def throttled_tasks(self):
                 #
                 # We should refactor the chart generator to make this classification logic more flexible so the user can specify
                 # which tasks / or types of operations should be used for which chart types.
-                if "target-throughput" in sub_task.params or "target-interval" in sub_task.params or sub_task.operation.type == "eql":
+                if "target-throughput" in sub_task.params or "target-interval" in sub_task.params\
```
Comment (danielmitterdorfer) [inline esrally/chart_generator.py:1669]: How about we instead use the operation type as an indicator? I propose the following operation types (in addition to `eql`):

* `search`
* `composite`
* `paginated-search`
* `scroll-search`

**Review by danielmitterdorfer — COMMENTED** [review]
Thanks! I left a suggestion how to categorize throttled tasks.

File: esrally/chart_generator.py (hunk 1 above)
Comment (ebadyano) [reply esrally/chart_generator.py]: hmm, just realize that if we only check for type of operation we will miss raw-requests like in https://github.com/elastic/rally-tracks/pull/179/files, I wonder if we still want to keep ` "target-throughput" in sub_task.params or "target-interval"` as well as type of operations?

Comment (danielmitterdorfer) [reply esrally/chart_generator.py]: I agree, we should also consider `target-throughput` and `target-interval` as before.

**Comment by ebadyano** [comment]
@danielmitterdorfer Is this okay to merge? Thank you

File: esrally/chart_generator.py [hunk 2]
```diff
@@ -1678,7 +1690,8 @@ def throttled_tasks(self):
                 # We should refactor the chart generator to make this classification logic more flexible so the user can specify
                 # which tasks / or types of operations should be used for which chart types.
                 if sub_task.operation.type == "search" or sub_task.operation.type == "composite" or sub_task.operation.type == "eql"\
-                    or sub_task.operation.type == "paginated-search" or sub_task.operation.type == "scroll-search":
```
Comment (danielmitterdorfer) [inline esrally/chart_generator.py]: We can simplify the checks for the operation type:

```python
if sub_task.operation.type in ["search", "composite", "eql", "paginated-search", "scroll-search"] ...
```

**Review by danielmitterdorfer — COMMENTED** [review]
I left one suggestion but apart from that it looks fine.

**Review by danielmitterdorfer — APPROVED** [review]
Thank you! LGTM

**Pull request merged**